sdk/go/pulumi/types.go

```diff
@@ -389,11 +391,31 @@ func (o *OutputState) ApplyTWithContext(ctx context.Context, applier interface{}
 		}
 
 		// Fulfill the result.
-		result.fulfillValue(results[0], true, nil)
+		result.fulfillValue(results[0], true, secret, nil)
 	}()
 	return result
 }
 
+// IsSecret returns a bool representing the secretness of the Output
```

Comment:
Do we expose this in any other SDKs?

Reply:
Looks like I missed the `internal` annotation in the node sdk. Will make private. 

https://github.com/pulumi/pulumi/blob/7fff99d346f140d32a196aeaf1aab192d0e0791d/sdk/nodejs/output.ts#L41